src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,29 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    enum Color {
+        RED,
+        GREEN,
+        BLUE,
+        YELLOW,
+        ORANGE
+    }
+
+    private final int index = new Random().nextInt(Color.values().length);
```

Comment:
1. should be calculcated in separate method, not only once in class
2. random should be `private final` class-level field

Reply:
Can you please explain how to will be that "random should be private final class-level field", because I can't understand it at all.